CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+# Code of Conduct
+
+## Conduct
+### Contact: https://airtable.com/shrFiQnLrcpeBp2lS
```

Comment:
this links to defi wallet's contact

Reply:
It is the same as:
https://github.com/cronos-labs/play-unreal-demo/blob/main/README.md?plain=1#L45
https://github.com/cronos-labs/play-unreal-plugin/blob/main/README.md?plain=1#L52
Should it change? @ricky321q

Reply:
I think this link (https://airtable.com/shrFiQnLrcpeBp2lS) is the right form to contact to us.
What do you mean for links to defi wallet's contact? @tomtau

Reply:
oh, perhaps ok

Reply:
you can see below:

> Remarks that violate the Crypto.com standards of conduct, (...)

> In the Crypto.com developer community (...)

Reply:
maybe ok as is, it may be clear that the moderator is cronos labs